action.yml

```diff
@@ -16,7 +16,8 @@ branding:
 inputs:
   github_token:
     description: GITHUB_TOKEN with at least write access to pull-requests scope or a repo scoped PAT
-    required: true
+    # The token is not required if comment generation is skipped
+    required: false
     default: ${{ github.token }}
```

Comment:
The current action doesn't error if you don't provide a token because it's got a default value right here... The `required` setting changes nothing.

Reply:
True. The change was made for those people (me, sometimes) who read `action.yml` files as the "true" source of documentation for GitHub Marketplace actions. It felt more correct to make it clear there that the token is not always required. Even still, the change has been reverted, which also means no new tag/release is needed.